decisions/why-material-design/index.qmd

```diff
@@ -0,0 +1,44 @@
+---
+title: "Why Material Design"
+description: "We want to Seedcase to look good with a minimal effort."
+author: "Philip Christiansen"
+date: last-modified
+categories:
+---
+
+
+## Context and Problem Statement
+
+Seedcase should look good with a minimal effort. Therefore, we want to leverage existing CSS or UI frameworks rather than starting from scratch. 
+
+## Decision Drivers
+
+The UI framework should:
+- Should look good
+- Integrate well with figma. The wireframes in figma should be easy to reproduce in Django templates
+- Be easy to use with Django. Since Django is a python framework, we don't want to rely on frameworks that requires node/npm or too much javascript. Preferably, we want a CSS framework, where we can just add CSS classes to html elements
```

Comment:
```suggestion
- Be easy to use with Django. Since Django is a Python framework, we don't want to rely on frameworks that requires node/npm or too much JavaScript. Preferably, we want a CSS framework, where CSS classes can be directly added to HTML elements
```